app/views/help/cookie_settings.html.erb

```diff
@@ -1,13 +1,16 @@
 <% content_for :title, "Cookies on GOV.UK" %>
-<main id="content" role="main" class="group help-page cookie-settings govuk-grid-column-two-thirds">
-    <div class="cookie-settings__confirmation" data-cookie-confirmation="true">
+<main id="content" role="main" class="group help-page cookie-settings govuk-grid-row">
+  <div class="cookie-settings__confirmation govuk-grid-column-two-thirds" data-cookie-confirmation="true">
       <%= render "govuk_publishing_components/components/success_alert", {
         message: t('cookies.confirmation_title'),
         description: raw("<p class='govuk-body'>#{t('cookies.confirmation_message')}</p>
-        <a class='govuk-link govuk-notification-banner__link cookie-settings__prev-page' href='#' data-module='gem-track-click' data-track-category='cookieSettings' data-track-action='Back to previous page'>#{t('cookies.confirmation_previous_referrer_link')}</a>").html_safe
+        <a class='govuk-link govuk-notification-banner__link cookie-settings__prev-page' href='#' data-module='gem-track-click' data-track-category='cookieSettings' data-track-action='Back to previous page'>#{t('cookies.confirmation_previous_referrer_link')}</a>").html_safe,
+        margin_bottom: 0,
       } %>
   </div>
 
+  <div class="govuk-grid-column-two-thirds">
```

Comment:
Why did you add another grid column here? It's the same as the one above - would all of the content not fit into that one?